src/components/GettingStartedModal/ChooseSetStep.tsx

```diff
@@ -0,0 +1,110 @@
+import { ChangeEvent, FormEvent, ReactElement, useEffect } from "react";
+import { NavigationButtons, Step, StepProps } from ".";
+import { Button } from "../Button";
+import {
+  isPhoneticsPreference,
+  PhoneticsPreference,
+  PREFERENCE_LITERATES,
+} from "../../state/reducers/phoneticsPreference";
+import { GROUPS, isGroupId } from "../../state/reducers/groupId";
+import { SectionHeading } from "../SectionHeading";
+import { CollectionDetails } from "../CollectionDetails";
+
+import { collections, VocabSet } from "../../data/vocabSets"; 
+import { useUserStateContext } from "../../state/UserStateProvider";
+import { TermsByProficiencyLevelChart } from "../TermsByProficiencyLevelChart";
+
+export const ChooseSetStep: Step = {
+
+  
+  title: "Phonetics",
+  commitState: ({ collectionId }, {setUpstreamCollection}) => {
+    setUpstreamCollection(collectionId);
+  },
+  Component: PhoneticsStepComponent,
+};
+
+function PhoneticsStepComponent({
+  wizardState: { collectionId, groupId },
+  setWizardState,
+  goToNextStep,
+  goToPreviousStep,
+}: StepProps): ReactElement {
+  function setWizardStateCollectionId(
+    newCollectionId: string
+  ) {
+    setWizardState((s) => ({
+      ...s,
+      collectionId: newCollectionId,
+    }));
+  }
+  function onRadioChanged(e: ChangeEvent<HTMLInputElement>) {
+    const collectionId  = e.target.value;
+      setWizardStateCollectionId(collectionId);
+  }
+  function onSubmit(e: FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+    if (collectionId) goToNextStep();
+  }
+
+
+  function totalTerms(vocab: VocabSet[]){
+    var t = 0; 
+
+    vocab.map((vocabSet) => (
+      t += vocabSet.terms.length
+    ))
+
+    return t; 
+  } 
```

Comment:
This can be written with reduce as:
```ts
vocab.reduce((tot, set) => tot + set.terms.length, 0);
```

Or with `forEach` as:
```
let t = 0;
vocab.forEach(set => t += set.terms.length);
return t;
```

But `map` functions should _always_ be pure and return a result.

I would say the `reduce` version is better than `forEach` though.